backend/src/routes/api/nb-events/index.ts

```diff
@@ -0,0 +1,17 @@
+import { FastifyInstance } from 'fastify';
+import { getNotebookEvents } from './eventUtils';
+
+export default async (fastify: FastifyInstance): Promise<void> => {
+  fastify.get('/:notebookName', async (request, reply) => {
+    const params = request.params as {
+      notebookName: string;
+    };
+    return getNotebookEvents(fastify, params.notebookName)
+      .then((res) => {
+        return res;
+      })
+      .catch((res) => {
+        reply.send(res);
+      });
```

Comment:
I do not believe you need these... the fastify architecture should handle errors that are returned to it and `.then((v) => v)` functions that just pass a value through do nothing for Promises, those can always be omitted.

Reply:
Removed .then, left the .catch in to catch errors in Utils